<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exposuretime</class>
 <widget class="QDialog" name="Exposuretime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QLabel" name="ExposureDescriptionLabel">
       <property name="text">
        <string>min:s:ms:µs</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="DelayLabel">
       <property name="text">
        <string>Delay</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="DelayDescriptionLabel">
       <property name="text">
        <string>min:s:ms:µs</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="TimingLabel">
       <property name="text">
        <string>Timing</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="DelayCurrentValue">
       <property name="text">
        <string>00:00:000:000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="ExposureCurrentValue">
       <property name="text">
        <string>00:00:000:000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="DelayCurrentValueLabel">
       <property name="text">
        <string>Current Delay</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ExposureLabel">
       <property name="text">
        <string>Exposure</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="DelayLineEdit">
       <property name="inputMask">
        <string>00:00:000:000</string>
       </property>
       <property name="text">
        <string>00:00:000:000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="CurrentExposureLabel">
       <property name="text">
        <string>Current Exposure</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="ExposureLineEdit">
       <property name="inputMask">
        <string>00:00:000:000</string>
       </property>
       <property name="text">
        <string>00:00:000:000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="7">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CloseButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ApplyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Full setup is only available if the camera is not running!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CloseButton</sender>
   <signal>clicked()</signal>
   <receiver>Exposuretime</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
